cmd/minishift/cmd/stop.go

```diff
@@ -55,7 +55,15 @@ func runStop(cmd *cobra.Command, args []string) {
 		atexit.ExitWithMessage(0, fmt.Sprintf("The '%s' VM is already stopped.", constants.MachineName))
 	}
 
-	fmt.Println("Stopping local OpenShift cluster...")
+	fmt.Println("Stopping the OpenShift cluster...")
+
+	if hostVm.Driver.DriverName() == "generic" {
+		if err := util.OcClusterDown(hostVm); err != nil {
+			atexit.ExitWithMessage(1, err.Error())
+		}
+		fmt.Println("Cluster stopped.")
+		return
+	}
```

Comment:
`fmt.Println("Cluster stopped.")` is there in two places in the function.  One for `generic` driver and another for non generic driver. IMO this is a place to use `else` to keep the logic separated and also helps with one `fmt.Println("Cluster stopped.")`.